docs/staking-and-delegation/managing-stake-tao-app.md

```diff
@@ -2,13 +2,33 @@
 title: "Managing Stake with tao.app"
 ---
 
-[TAO.app](https://tao.app) offers a portal to real-time tokenomic data about the Bittensor ecoystems and all its subnets, and allows users to securely manage their stake using the Bittensor wallet browser extension.
+[Tao.app](https://tao.app) offers a portal to real-time tokenomic data about the Bittensor ecoystems and all its subnets, and allows users to securely manage their stake using the Bittensor wallet browser extension.
+
+At the very top of the main page, there are several metrics which gives some of the most important information an ecosystem user would likely want to be informed with:
+
+* **TAO Price**: The current price of the TAO coin and percentage increase or decrease over the last day.
```

Comment:
* **TAO Price**: The current price of $TAO in US Dollar and percentage change over the last day.